docs/stacks-guide.md

```diff
@@ -0,0 +1,301 @@
+
+* [Crossplane concepts][crossplane-concepts]
+* [Kubernetes concepts][kubernetes-concepts]
+
+## Before you get started
+
+This guide assumes you are using a *nix-like environment. It also assumes you have a basic working familiarity with the following:
+
+* The terminal environment
+* Setting up cloud provider accounts for the cloud provider you want to
+  use
+* [Kubernetes][kubernetes-docs] and [kubectl][kubectl-docs]
+
+You will need:
+
+* A *nix-like environment
+* A cloud provider account, for the cloud provider of your choice (out
+  of the supported providers)
+* A locally-configured kubectl which points to a configured Kubernetes
+  cluster. We will put Crossplane in this cluster, and we'll refer to it
+  as the control cluster.
+
+## Install the Crossplane CLI
+
+To interact with stacks, we're going to use the [Crossplane
+CLI][crossplane-cli], because it's more convenient. To install it, we
+can use the one-line curl bash:
+
+```
+RELEASE=0.0.1
+curl -sL https://raw.githubusercontent.com/crossplaneio/crossplane-cli/"${RELEASE}"/bootstrap.sh | bash
+```
+
+To use the latest release, you can use `master` as the `RELEASE` instead
+of using a specific version.
+
+## Install and configure Crossplane
+
+To use Crossplane, we'll need to install and configure it. In this case,
+we want to use Crossplane with a cloud provider, so we'll need to
+configure the provider.
+
+### Install Crossplane
+
+The recommended way of installing Crossplane is by using
+[helm][helm-install]. We can grab the most stable version currently
+available by using:
+
+```
+helm repo add crossplane-alpha https://charts.crossplane.io/alpha
+helm install --name crossplane --namespace crossplane-system crossplane-alpha/crossplane
+```
+
+For more options for installing, including how to install a more
+bleeding-edge version, or how to uninstall, see the [full install
+documentation][crossplane-install-docs].
+
+### Configure support for your cloud provider
+
+Next we'll set up support for our cloud provider of choice! See the
+provider-specific guides:
+
+* [AWS][aws-setup]
+* [GCP][gcp-setup]
+* [Azure][azure-setup]
+
+Then come back here! Don't worry; we'll still be here when you're ready.
+
+Don't see your favorite cloud provider? [Help us add
+support][provider-stack-developer-guide] for it!
+
+## Install support for our application into Crossplane
+
+Now that we've got Crossplane set up and configured to use a cloud
+provider, we're ready to add support for creating WordPresses! We'll do
+this using a Crossplane Stack. For more information about stacks, see
+the [full Stack documentation][stack-docs].
+
+We can use the [Crossplane CLI][crossplane-cli] to install our stack which adds support for
+Wordpress:
+
+```
+kubectl crossplane stack install 'crossplane/sample-stack-wordpress:latest' 'sample-stack-wordpress'
```

Comment:
We will need to make sure that the wordpress stack is installed in the same `namespace` as the portable classes created in each individual stack guide. We could either standardize on the namespace or just mention that it needs to be the same here. I used this to install wordpress stack into the namespace of my portable classes:

```
kubectl crossplane stack generate-install  'crossplane/sample-stack-wordpress:latest' 'sample-stack-wordpress' | kubectl apply --namespace app-project1-dev -f -
```

Reply:
This should be fixed now